<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectManipulatorDlg</class>
 <widget class="QWidget" name="DirectManipulatorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>273</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DirectManipulatorDlg</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>视点经度</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>视点纬度</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>视点高度</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>旋转角度</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>目标经度</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>目标纬度</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>目标高度</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_EyeLon"/>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_EyeLat"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_EyeHei"/>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_Angle"/>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_TargetLon"/>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="lineEdit_TargetLat"/>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="lineEdit_TargetHei"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_Set">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Revert">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
